Kubernetes/legos/k8s_get_pods_in_not_running_state/k8s_get_pods_in_not_running_state.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2023 unSkript.com
+# All rights reserved.
+#
+from typing import Tuple
+from kubernetes import client
+from kubernetes.client.rest import ApiException
+from pydantic import BaseModel, Field
+
+
+class InputSchema(BaseModel):
+    namespace: str = Field(
+        '',
```

Comment:
if you dont specify namespace, does it pick it up from default?

Reply:
If not specified, all namespaces will be used.